empress_cli.py

```diff
@@ -6,6 +6,10 @@
 
 import argparse
 
+import matplotlib
+# use agg backend, see # https://matplotlib.org/faq/usage_faq.html#what-is-a-backend
+matplotlib.use("agg")
```

Comment:
Let's open an issue that talks about how tkinter backend breaks your computer. Then, write another line of comment which says "agg" is a non-interactive backend which cannot pop up a window
```python
# "agg" is a non-interactive backend which cannot pop up a window
# we don't need to pop up a window in the clip, there is a tkinter bug which occurs when we 
# pop up a matplotlib window from the cli.
# see [link to the issue]
```

Reply:
Actually let's just remove the whole thing and document it in the pull request description.